docs/dev/README.md

```diff
@@ -0,0 +1,37 @@
+# Developing the Glean SDK
+
+![Glean logo](glean.jpeg)
+
+The `Glean SDK` is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+To contact us you can:
+- Find us in the [#glean channel on chat.mozilla.org](https://chat.mozilla.org/#/room/#glean:mozilla.org).
+- To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
+- Send an email to *glean-team@mozilla.com*.
+- The Glean SDK team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
+
+The source code is available [on GitHub](https://github.com/mozilla/glean/).
+
+## Using this book
+
+This book is specifically about **contributing to and developing** the `Glean SDK`.
+
+For documentation on **using** the `Glean SDK`, refer to [the Glean Book](https://mozilla.github.io/glean/).
+
+For documentation about the broader end-to-end Glean project, refer to the [Mozilla Data Documentation](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+This book is divided into 2 main parts:
```

Comment:
Yeap, honestly I only changed the content of these files in this PR so that they would make sense. I agree both pages should be edited. I'll file a follow up for that right away.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1693489